makefile

```diff
@@ -157,13 +157,15 @@ apidocs:
 	cp doc/doxygen.css api_docs/html/
 	@echo "\n=> Open up 'api_docs/html/index.html' in a browser to view a local copy of the documentation"
 
-STYLIZE_DIFFBASE ?= staging
+# Find the most recent common ancestor. This prevents new commits on staging from registering as diffs.
+DIFFBRANCH ?= staging
```

Comment:
I kept this as staging to avoid breaking peoples' workflows if they use `upstream` or something different instead of `origin`.